src/nvim/lua/stdlib.c

```diff
@@ -532,4 +532,8 @@ void nlua_state_add_stdlib(lua_State *const lstate)
 
   lua_cjson_new(lstate);
   lua_setfield(lstate, -2, "json");
+
+  // TODO(lewis6991): Remove after 0.7 release
+  lua_pushboolean(lstate, 1);
```

Comment:
our policy is to not do things like this. master-following plugins should expect the user is also following master.

Reply:
Sure. Was just using this to dog food https://github.com/lewis6991/gitsigns.nvim/pull/438 without having to segregate a different setup.